roiextractors/extraction_tools.py

```diff
@@ -39,3 +43,92 @@ def get_video_shape(video):
         num_channels, num_frames, size_x, size_y = video.shape
 
     return num_channels, num_frames, size_x, size_y
+
+
+def check_get_frames_args(func):
+    @wraps(func)
```

Comment:
Let's talk about this. I don't understand why it is necessary.

Reply:
Hi Ben,
I think this is very handy for the following (avoiding to repeat code)

1. Check that all frames are < get_num_frames()
2. Handle single frame (previous `get_frame`) and multiple frames.
3. Make `frame_idxs` and array always (otherwise all functions would need to parse it independently

If in the future we need more checks, I think that is the right way to do it. Not necessary, but very handy